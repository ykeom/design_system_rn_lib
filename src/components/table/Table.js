import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Table extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>table</Text>
      </View>
    );
  }
}

export default Table;
