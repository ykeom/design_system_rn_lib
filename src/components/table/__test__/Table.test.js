import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Table from '../Table';

describe('Table', () => {
  it('should render', () => {
    const component = shallow(<Table />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
