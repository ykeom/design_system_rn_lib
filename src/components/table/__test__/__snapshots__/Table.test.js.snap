// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    table
  </Text>
</View>
`;
